src/components/VaultRow/lib/VaultDepositForm.tsx

```diff
@@ -34,31 +32,39 @@ import {
   InvalidMinAmountModal,
 } from "@/components/Modal"
 import TokenForm, { TokenFormValues } from "@/components/TokenForm/TokenForm"
+import { VaultRowPropsWithProduct } from "@/components/VaultRow/VaultRow"
 
 import { useGlobalStore } from "@/store"
 
-export const VaultDepositForm: FC<VaultProps> = (props) => {
+export type VaultDepositWithdrawProps = VaultRowPropsWithProduct & {
+  inputToken: Address
+  outputToken: Address
+  underlyingAssets: Address[] | readonly Address[] | undefined
+}
+
+export const VaultDepositForm: FC<VaultDepositWithdrawProps> = ({
+  inputToken,
```

Comment:
These names are a little misleading I think because at least for the inputToken, it's controlled within this component. Maybe `defaultInputToken` and `defaultOutputToken` would be better -- but I don't think those either really truly describe what is being passed in here.

Reply:
I think default one name would be better too.